developers.diem.com/docs/welcome-to-diem.mdx

```diff
@@ -102,34 +102,6 @@ no_pad_top: true
   />
 </CardsWrapper>
 
-<br/>
-
-## **Write Smart Contracts**
-
-<CardsWrapper cardsPerRow={3} justify={true}>
```

Comment:
Maybe keep these but link to the off-site docs?

Reply:
I had some discussions about this previously. At least at the time, the Move team wanted to have flexibility to rearrange the documentation in the future without needing to preserve links into the middle of the book, and so our proposal to folks at the Diem Association was to only have one link to the book as a whole.

Reply:
I discussed this with @tnowacki and we agreed it would be OK to keep these 3 links into the Move docs. This will also solve the problem of how to direct people to the Move docs, at least as a starting point. I've revised the PR to keep these 3 cards.